app/javascript/Snackbar/Snackbar.jsx

```diff
@@ -70,6 +74,24 @@ export class Snackbar extends Component {
           snack.lifespanTimeoutId = setTimeout(() => {
             this.decreaseLifespan(snack);
           }, 1000);
+
+          if (snack.addCloseButton) {
```

Comment:
From @benhalpern 

> On narrow article show views does it make sense to lift the snackbar above the reactions bar so it doesn't block what the user wants to do?
> 

@benhalpern, I think we could do that, but it is specific just to the article page. We can do some stuff to make that happen, but another option I thought of was allowing for a close button. I've added this as an optional property to snackbar items, called `addCloseButton`. Thoughts? Also, @ludwiczakpawel your thoughts?

![image](https://user-images.githubusercontent.com/833231/80777911-b6397300-8b34-11ea-8909-cd1ae82a185e.png)

![image](https://user-images.githubusercontent.com/833231/80777931-c4878f00-8b34-11ea-9713-9f5f467d52f4.png)

Reply:
+1 for the close buttons

Reply:
Yea, I think close button makes perfect sense. Maybe "Dismiss" as label would work better here? (stealing from gmail)... or even simple:

```
<Button variant="ghost" icon={Icon} contentType="icon-left" />
```
...with icon x.svg ?